Bio/PDB/Chain.py

```diff
@@ -7,10 +7,13 @@
 
 """Chain class, used in Structure objects."""
 
+from Bio.PDB.Atom import Atom
+from Bio.KEGG import Gene
+from Bio.PDB.Residue import Residue
 from Bio.PDB.Entity import Entity
 from Bio.PDB.internal_coords import IC_Chain
 
-from typing import Optional
+from typing import Generator, List, Optional
```

Comment:
It seems that both `typing.Generator` and `typing.List` will be deprecated soon (Python 3.9). See [here](https://docs.python.org/3/library/typing.html)